psh/test-kill.py

```diff
@@ -0,0 +1,145 @@
+# Phoenix-RTOS
+#
+# phoenix-rtos-tests
+#
+# psh kill command test
+#
+# Copyright 2021 Phoenix Systems
+# Author: Damian Loewnau
+#
+# This file is part of Phoenix-RTOS.
+#
+# %LICENSE%
+#
+
+from psh.tools.basic import run_psh, assert_only_prompt
+from time import sleep
+
+PROMPT = r'\r\x1b\[0J' + r'\(psh\)% '
+EOL = r'(\r+)\n'
+
+
+def is_hardware_target(p):
+    cmd = 'ls'
+
+    p.sendline(cmd)
+    p.expect(cmd)
+    idx = p.expect([
+            'syspage',
+            'bin'])
```

Comment:
Why not in one line?
```suggestion
    idx = p.expect(['syspage', 'bin'])
```